packages/dataviews/src/types.ts

```diff
@@ -356,6 +356,11 @@ export interface ViewTable extends ViewBase {
 		 * The density of the view.
 		 */
 		density?: Density;
+
+		/**
+		 * Switch on/off hierarchical sorting.
+		 */
+		hierarchicalSort?: boolean;
```

Comment:
If this is a sort, should it be part of `view.sort` instead and layouts could decide to apply it or ignore it.

Reply:
I don't know that this is a sort, to be honest, it's more like a "group"? Happy to change the name to `groupByHierarchy`, `displayHierarchy`, `hierarchy` or something along those lines.

Reply:
Actually, I'm not even sure this should be named `hierarchy` because the only thing this does is to display the `levels`. The order of the dataset and any other thing is done by the consumer (via REST API).

Perhaps `displayHierarchyLevel` would be a better name.

Reply:
My opinion is that we should remove `layout` sub key entirely, I don't think it's achieving what it was built for.

Reply:
ok, renamed it to `showLevels` and made it part of the view. I think a potential follow-up PR would be to implement similar changes to the list view — which eases my concerns of having something view-specific in the view config.